Corona-Warn-App/src/main/java/de/rki/coronawarnapp/srs/ui/symptoms/intro/SrsSymptomsIntroductionFragment.kt

```diff
@@ -106,7 +108,7 @@ class SrsSymptomsIntroductionFragment : Fragment(R.layout.fragment_submission_sy
 
         binding.symptomButtonNext.apply {
             isEnabled = symptomIndication != null
-            setText(
+            defaultButton.setText(
```

Comment:
button should be disabled until user selects at least one option, it is currently enabled by default If you click it without any selection, it keeps loading forever

Reply:
Fixed it.